docker-compose.yml

```diff
@@ -8,15 +8,16 @@ volumes:
   policies:
   vault:
   caches:
+  rstuf-worker-data:
+  rstuf-metadata:
 
 services:
   vault:
     # NOTE: pinned for consistency with whats available in our deployment
     image: vault:1.12.3
     restart: on-failure
     entrypoint: /bin/sh
-    command: /etc/vault/entry.sh
-    stop_signal: SIGINT
```

Comment:
The `command` rewrites and `stop_signal` removals  (here and below) seem unrelated. Can we remove them from this PR?

Reply:
I had an outdated fork. Thanks.